streams/tests/test_dask.py

```diff
@@ -54,6 +55,33 @@ def test_map(c, s, a, b):
 
 
 @gen_cluster(client=True)
+def test_compute(c, s, a, b):
+    source = Stream()
+
+    @delayed
+    def inc1(x):
+        """Immediate delayed function"""
+        return inc(x)
+
+    def inc2(x):
+        """Function that just returns a delayed"""
+        return delayed(inc)(x)
+
+    L1 = source.to_dask().compute(inc1).sink_to_list()
+    L2 = source.to_dask().compute(inc2).sink_to_list()
```

Comment:
I have an already implemented function which creates the following dag from an input record:

![image](https://cloud.githubusercontent.com/assets/961747/25063654/fc3aeb38-21ea-11e7-8c7c-fdb7a71e53e3.png)

Theoretically I could create the same `streaming DAG` but just computing a delayed is easier to start with.